artemis-server/pom.xml

```diff
@@ -89,6 +89,10 @@
          <version>${project.version}</version>
          <scope>test</scope>
       </dependency>
+      <dependency>
```

Comment:
Need to ensure this is updated in the OSGI / Karaf bundles config to ensure its bundled and loaded correctly, else those running in Karaf / OSGI they will get issue.

Reply:
I had to add this on features on my branch to be able to run tests.

Reply:
You have to add:

``xml
<bundle dependency="true">mvn:org.jctools/jctools-core/${jctools.version}</bundle>
```
 as a 
dependency on artemis-core. see my other comment